WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -33,7 +33,7 @@
           <tr>
             <td>
               <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
-              <br /><strong><%= competition.country_name %></strong>, <%= competition.cityName %>
+              <br /><strong><%= competition.country.name_in(:en) %></strong>, <%= competition.cityName %>
```

Comment:
Right, `cityName` is submitted by the organizer so it may be either in his native language or English.
But it's a bit different as there is no way to localize it.

Would you rather use `country.name` instead? I don't think it makes a big difference in an admin view, so whatever you prefer :)